exp/capacity-planning.md

```diff
@@ -0,0 +1,96 @@
+When planning your Anbox Cloud deployment, you should start by estimating how much capacity you need to be able to provide your application to your users and how many users you expect. Based on this estimate, you can then size your deployment and figure out how many cluster nodes you need and what resources they should have.
```

Comment:
`...and how many users you expect.`

as of now we don't have a concrete concept of `users`  being used for the sizing of deployment.
It feels more natural like `how many containers you can expect running simultaneously`

Reply:
We should make clear that when we talk about users we mean a 1:1 mapping between a user and a container. Then it becomes clear.